README.md

```diff
@@ -1,2 +1,8 @@
-# scci-core-services-api
-API definitions of the SCCI Platform core services
+# rokwire-core-services-api
+API definitions of the Rokwire Platform core services
+
+## How to view API design doc 
```

Comment:
In order to view the API definitions, a non-developer can go to Swagger UI demo page (https://petstore.swagger.io/) and paste our YAML file URL there (https://raw.githubusercontent.com/rokwire/scci-core-services-api/develop/rokwire.yaml). They don't have to sign up for an account.

Reply:
Added